lib/Command/ACL.php

```diff
@@ -117,7 +117,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$mappingType = $input->getOption('user') ? 'user' : 'group';
 			$mappingId = $input->getOption('user') ?: $input->getOption('group');
 			$path = $input->getArgument('path');
-			$path = trim($path, '/');
+			$path = trim((string)$path, '/');
```

Comment:
Maybe instead of casting we should check the content?
What the current type here? `string|null`? Is `(string)null === ''`? If so, then all good.